src/classes/DatabaseManager.js

```diff
@@ -33,49 +33,54 @@ class DatabaseManager {
 
   /**
    * Insert a guild into the guild_settings table.
-   * @param  {String}             id     The ID of the guild.
-   * @param  {String}             prefix The default prefix
-   * @return {(Number|undefined)}        Returns 0 on success or undefined.
+   * @param  {String}            id     The ID of the guild.
+   * @param  {String}            prefix The default prefix
+   * @return {Promise<Object[]>}        Returns the updated table.
    */
   async addClient (id, prefix) {
-    return this._qb.run('insert', { table: 'guild_settings', data: { id, prefix } })
-      .then(() => this._qb.run('insert', { table: 'guild_toggles', data: { id } }))
+    return this._qb.insert({ table: 'guild_settings', data: { id, prefix } })
+      .then(() => this._qb.insert({ table: 'guild_toggles', data: { id } }))
   }
 
   /**
    * Insert a status into the statuses table.
-   * @param  {String}             name   The name of the status.
-   * @param  {Number}             [type] The type of the status.
-   * @return {(Number|undefined)}        Returns 0 on success or undefined.
+   * @param  {String}            name   The name of the status.
+   * @param  {Number}            [type] The type of the status.
+   * @param  {Boolean}           [def]  Whether or not this is the default status.
+   * @return {Promise<Object[]>}        Returns the updated table.
    */
   addStatus (name, type, def) {
-    return this._qb.run('insert', { table: 'statuses', data: { name, type, default: def } })
+    return this._qb.insert({ table: 'statuses', data: { name, type, default: def } })
   }
+  /**
+   * Grabs default statuses for a guild
```

Comment:
that's not even close to what this function does. this function returns the default presence of the bot.

Reply:
Lmao

Reply:
well uhhh... sorry okay